jmeter/src/main/java/com/focusit/jmeter/JMeterScriptProcessor.java

```diff
@@ -91,29 +93,30 @@ public boolean processSampleDuringRecord(HTTPSamplerBase sampler, SampleResult r
         }
         else
         {
-            log.error(Thread.currentThread().getName() + ":" + "Sample " + sampler.getName()
+            LOG.warn(Thread.currentThread().getName() + ":Sample " + sampler.getName()
```

Comment:
Why? Our program doesn't break, if we come to this branch of code. We simply notify, that this script doesn't work as expected